complete.py

```diff
@@ -267,42 +268,70 @@ def _add_V_step(self, force_group="0"):
 
         # update velocities
         if self._mts:
-            self.addComputePerDof("v", "v + ((dt / {}) * moddi / m)".format(self._force_group_nV[force_group], force_group))
+            self.addComputePerDof("v", "v + ((dt / {}) * modified_force / m)".format(self._force_group_nV[force_group], force_group))
         else:
-            self.addComputePerDof("v", "v + (dt / {}) * moddi / m".format(self._force_group_nV["0"]))
+            self.addComputePerDof("v", "v + (dt / {}) * modified_force / m".format(self._force_group_nV["0"]))
 
         self.addConstrainVelocities()
 
 
         self.addComputeSum("new_ke", self._kinetic_energy)
         self.addComputeGlobal("shadow_work", "shadow_work + (new_ke - old_ke)")
-    
+
     def _add_F_step(self):
         """
-        add an moddi update step
+        pull the mm force as a perDofVariable from the integrator to be modified
         """
-        self.addComputePerDof('moddi', 'f')
-        
-    
+        self.addComputePerDof('modified_force', 'f')
+
+
     def _add_variables(self):
         super()._add_variables()
-        self.addPerDofVariable('moddi', 0)
-    
+        self.addPerDofVariable('modified_force', 0)
+
     def _add_integrator_steps(self):
         """
         Add the steps to the integrator--this can be overridden to place steps around the integration.
         """
         super()._add_integrator_steps()
         #self.addUpdateContextState()
-    
+
     @property
     def _step_dispatch_table(self):
         dispatch_table = super()._step_dispatch_table
-        dispatch_table['F'] = (self._add_F_step, False) #add a moddi variable
+        dispatch_table['F'] = (self._add_F_step, False) #add a modified_force variable
         return dispatch_table
-        
-    
+
+
 class Propagator(OMMBIP):
+    """
+    Propagator pseudocode:
+    Step 1: initialization--
+        set iteration = 0, n_iterations = n_iterations, lambda  = 0 (i.e. iteration / n_iterations); work_accumulated = 0.0
+        generate sample x_0 ~ e^(-p(x))
```

Comment:
defining "p" as reduced potential threw me for a loop -- pretty ingrained to think of "p" as a normalized density or similar

Reply:
i am very inconsistent; it is treated like a reduced potential; can change